### PR TITLE
Don't show empty lines, open catchments or 'referred by self' in elig…

### PR DESCRIPTION
**Pull request by DanielHeath**
…ibility

### CODE REVIEW

**Comment by DanielHeath** [comment]
@kane-c ++?

File: src/components/Eligibility.js [hunk 1]
```diff
@@ -16,37 +16,38 @@ class Eligibility extends React.Component {
...
+                        {eligibleItems}
                     </ul>
                 </div>
             );
         }
 
-        if (!_.isEmpty(ineligibilityInfo)) {
+        let ineligibleItems = this.renderEligibility(
+            this.props.ineligibility_info
+        )
```
Comment (kane-c) [inline src/components/Eligibility.js]: `;`

File: src/components/Eligibility.js [hunk 2]
```diff
@@ -64,34 +65,32 @@ class Eligibility extends React.Component {
...
+            .toLocaleLowerCase().trim();
 
-        if (!_.isEmpty(referralInfo)) {
-            return (
-                <li>Referred by {referralInfo}</li>
-            );
-        } else {
-            return null;
+        if (!referralInfo.match(/^self\.?$/i)) {
+            return this.renderItem(`Referred by ${referralInfo}`)
```
Comment (kane-c) [inline src/components/Eligibility.js]: `;`

**Comment by kane-c** [comment]
Does the linter not check for semicolons?

**Comment by DanielHeath** [comment]
Odd - it usually does.

**Comment by DanielHeath** [comment]
@kane-c anything else?

**Comment by kane-c** [comment]
:+1:

**Pull request merged**